app/models/concerns/search.rb

```diff
@@ -0,0 +1,10 @@
+module Search
+  extend ActiveSupport::Concern
+  module ClassMethods
+    def search(query)
+      lambda do |args|
+        where(query, "%#{args}%", "%#{args}%", "%#{args}%")
+      end
```

Comment:
Codeclimate states that the code is identical or similar - https://codeclimate.com/github/Kotany-IF070-WebUI-RoR/NewVasjuky/pull/124

So thats why i moved it to concern

Now i refactored as i you said, but codeclimate check fails because of it